modules/ROOT/pages/log-points.adoc

```diff
@@ -1,17 +1,92 @@
 = Anypoint Monitoring Log Points
 
-The Anypoint Monitoring log points feature enables you to interactively extract data from running managed APIs. You can specify properties (for example, the host and method in the request header) in your managed API for which to generate logs without having to write code or redeploy your application. This helps you troubleshoot without having to restart the application. 
+== Log Points for Apps
+
+The Log Points feature for apps enables you to log information from connectors in any Mule app in real time. If you are using one of the supported connectors, you can select a log level (debug, info, warn, or error) and generate logs. 
+
+=== Prerequisites
+
+* A https://www.mulesoft.com/anypoint-pricing[Titanium subscription] to Anypoint Monitoring and Mule 3.8.5 or later
+* The *Runtime Manager > Manage Settings* permission
+
+=== Supported Connectors
```

Comment:
I think a three-column table called Supported Connectors might work. Here is an example, where you could put a checkmark or something similar in a cell to indicate that a connector is supported. (I don't know the key sequence to do a check mark).

Connector  |  Mule 4x. | Mule 3x
Anypoint Connector for Amazon S3 (Amazon S3) | x |  x

Use the naming convention above for most connectors. I am not sure about the naming conventions for the DB, Email, or Object Store connectors. Mongo should be MongoDB

Reply:
Oh, I like that idea.